src/MongoDB.Driver.Core/Core/WireProtocol/UpdateWireProtocol.cs

```diff
@@ -29,7 +28,7 @@ internal class UpdateWireProtocol : WriteWireProtocolBase
         private readonly bool _isMulti;
         private readonly bool _isUpsert;
         private readonly BsonDocument _query;
-        private readonly BsonDocument _update;
+        private readonly BsonValue _update;
```

Comment:
Undo all changes to this file.

Reply:
This class corresponds to the OP_UPDATE message and OP_UPDATE is not being changed to support pipeline updates.